### PR TITLE
Add rubygem-activerecord-nulldb-adapter package

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: packages/foreman/rubygem-activerecord-nulldb-adapter/rubygem-activerecord-nulldb-adapter.spec [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+%files
+%dir %{gem_instdir}
+%exclude %{gem_instdir}/.gitignore
+%exclude %{gem_instdir}/.travis.yml
+%{gem_instdir}/Appraisals
```
Comment (tbrisker) [inline packages/foreman/rubygem-activerecord-nulldb-adapter/rubygem-activerecord-nulldb-adapter.spec:75]: I think this should be `exclude` or moved to doc (it's used by ci testing)

File: packages/foreman/rubygem-activerecord-nulldb-adapter/rubygem-activerecord-nulldb-adapter.spec [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+%{gem_instdir}/CHANGES.md
```
Comment (tbrisker) [inline packages/foreman/rubygem-activerecord-nulldb-adapter/rubygem-activerecord-nulldb-adapter.spec:76]: `doc`?

File: packages/foreman/rubygem-activerecord-nulldb-adapter/rubygem-activerecord-nulldb-adapter.spec [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+%license %{gem_instdir}/LICENSE
+%{gem_instdir}/gemfiles
```
Comment (tbrisker) [inline packages/foreman/rubygem-activerecord-nulldb-adapter/rubygem-activerecord-nulldb-adapter.spec:78]: `exclude` or move to `doc`?  these are used by ci testing.

File: packages/foreman/rubygem-activerecord-nulldb-adapter/rubygem-activerecord-nulldb-adapter.spec [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+%{gem_instdir}/Gemfile
+%doc %{gem_instdir}/README.rdoc
+%{gem_instdir}/Rakefile
+%{gem_instdir}/activerecord-nulldb-adapter.gemspec
+%{gem_instdir}/spec
+
+%changelog
+* Sun Apr 19 2020 Eric D. Helms <ericdhelms@gmail.com> 0.4.0-1
+- Update to 0.4.0-1
+
```
Comment (tbrisker) [inline packages/foreman/rubygem-activerecord-nulldb-adapter/rubygem-activerecord-nulldb-adapter.spec:94]: Looks like an extra entry?

**Comment by ehelms** [comment]
Thanks for the finds. Updated and added a commit to include with the scl template.

**Comment by ehelms** [comment]
[test rpm]

**Pull request merged**